### PR TITLE
#210 (UNLV-FIA Homepage) Add Stacey Lai Profile to Team Page. (Based On main)

### PR DESCRIPTION
**Pull request by Lais3**
This is for issue #210 

## Code Changes
- added Stacey alphabetically
- ensured image size, ids, and no disruption to existing employees

### CODE REVIEW

**Review by michellemcgowan — APPROVED** [review]
@Lais3 nice work! Everything looks good!

**Review by ballec6 — APPROVED** [review]
Looks good!

**Review by Rosadc1 — APPROVED** [review]
Profile looks good Stacey!

File: TeamPage.html [hunk 1]
```diff
@@ -566,6 +566,55 @@ <h2>Meet Our Team</h2><br>
...
+                    <div class="modal-header">
+                      <div class="modal-title">
+                        <p class="text-justify"><b>Stacey Lai</b> - Student Programmer<br>
+                          Work Location: Remote<br>
+                          Email: stacey.lai@unlv.edu</p>
+                      </div>
+                      <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
+                    </div>
+                    <div class="modal-body">
+                      <p class="text-justify">Stacey Lai is in her 3rd year working towards a B.S. computer science degree, studying at
```
Comment (Kelvyn-Meyers) [inline TeamPage.html:593]: Minor suggestion here if you see it appropriate:
> ... towards a B.S. in Computer Science degree, ...

**Review by Kelvyn-Meyers — CHANGES_REQUESTED** [review]
Hi @Lais3,

Good work adding your profile to the website! I had one small suggestion for the first sentence of your biography. If you agree with it, feel free to implement the change and I can get this merged in. Let me know if you have any questions!

File: TeamPage.html (hunk 1 above)
Comment (Lais3) [reply TeamPage.html]: @Kelvyn-Meyers Hi! It's fixed now!

**Review by Kelvyn-Meyers — APPROVED** [review]
Very nice work here @Lais3! You resolved my request and your profile has been successfully added to the website. Ready to merge!

**Pull request merged**